### PR TITLE
Allwinner: Add support for Tanix TX6

### PR DESCRIPTION
**Pull request by jernejsk**
While this was specifically tested on Tanix TX6 STB, it also works on other, similarly designed STBs.

### CODE REVIEW

File: projects/Allwinner/devices/H6/patches/linux/12-ac200-nodes.patch [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
++			interrupts = <GIC_SPI 7 IRQ_TYPE_LEVEL_HIGH>;
++			clocks = <&ccu CLK_BUS_I2C3>;
++			resets = <&ccu RST_BUS_I2C3>;
++			pinctrl-names = "default";
++			pinctrl-0 = <&i2c3_pins>;
++			status = "disabled";
++			#address-cells = <1>;
++			#size-cells = <0>;
++
++			ac200: mfd@10 {
```
Comment (clementperon) [inline projects/Allwinner/devices/H6/patches/linux/12-ac200-nodes.patch]: this should be moved to the board device-tree

Comment (jernejsk) [reply projects/Allwinner/devices/H6/patches/linux/12-ac200-nodes.patch]: why? ac200 chip is co-packaged with H6, so it's inseparable. Only question remains if board is using any functionality of ac200 or not. Many LE relevant boards will use analog codec from it, when driver will be available.

Comment (clementperon) [reply projects/Allwinner/devices/H6/patches/linux/12-ac200-nodes.patch]: Oups sorry didn't know that the H6 had a co-packaged ac200

Comment (jernejsk) [reply projects/Allwinner/devices/H6/patches/linux/12-ac200-nodes.patch]: This chip provides many things - ephy, analog codec, tv out, rtc and efuse, although efuse seems to be unused in H6 case.

**Pull request merged**